dev.php

```diff
@@ -187,6 +187,13 @@ function shortcode_ui_dev_advanced_example() {
 			'multiple' => true,
 		),
 		array(
+			'label'    => esc_html__( 'Select Term', 'shortcode-ui-example' ),
+			'attr'     => 'term',
+			'type'     => 'term_select',
+			'taxonomy' => 'post_tag',
+			'multiple' => true,
+		),
+		array(
 			'label'  => esc_html__( 'Background Color', 'shortcode-ui-example' ),
```

Comment:
That one was already in there. If I change that one then I might as well change all of the `dev.php` file. Reckon I should do that then?